src/python/grpcio/grpc/__init__.py

```diff
@@ -170,7 +170,8 @@ def traceback(self, timeout=None):
 
     @abc.abstractmethod
     def add_done_callback(self, fn):
-        """Adds a function to be called at completion of the computation.
+        """Adds a function to be called at completion of the computation,
+        whether or not the computation is terminated sucessfully.
```

Comment:
["A doc string should be organized as a summary line (one physical line) terminated by a period, question mark, or exclamation point, followed by a blank line, followed by the rest of the doc string starting at the same cursor position as the first quote of the first line."](https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments)

This is good content to add; it just doesn't fit into the one-line summary. Put it in one of the arbitrary-length paragraphs that follows the one-line summary.

Reply:
Thanks. Done